index.md

```diff
@@ -8,6 +8,356 @@ This is some welcome text.
+import math  # importing a module twice
+```
+
+
+### W0401: Wildcard import {#W0401}
+
+Only import what you need. Wildcard imports are generally bad because they
+'pollute' the namespace with all the functions. For example:
+
+```python
+from valid_module import *
+```
+
+Rather than importing everything with wildcard '*', try importing specific
+functions for example:
+
+```python
+from module_name import certain_definition
+```
+
+Or, if you need to import many functions, import them the following way, to
+keep them separated in the namespace by module name. Then you can use the
+function like, module_name.function_name
+
+```python
+import module_name
+```
+
+Or, you can create an alias for module names in the way below. Which can be
+used like: dino.function_name
+
+```python
+import tyrannosaurus_rex as dino
+```
+
+
+### C0411: Wrong import order {#C0411}
+
+Used when PEP8 import order is not respected (do the standard imports first,
+then third-party libraries, then local imports)
+
+```python
+import your_own_module  # your own modules should be imported last.
+import sys  # "standard modules" should be imported first
+```
+
+
+### C0413: Wrong import position {#C0413}
+
+Imports should be placed at the top of the module (below the docstring)....
+
+```python
+my_list = ['a', 'b']
+import math  # this import should be at the top.
+```
+
+
+### C0412: Ungrouped imports {#C0412}
+
+Imports should be grouped by packages.
+
+```python
+from sys import byteorder  # same packages should be grouped
+from math import floor
+from sys import stdin  # same packages should be grouped
+```
+
+
+### C0410: Multiple imports {#C0410}
+
+Don't import multiple modules on one line.
+
+```python
+import sys, math
+```
+
+Do this instead:
+
+```python
+import sys
+import math
+```
+
+Although, note that you can import multiple functions on a line, when they are
+from the same module, for example:
+
+```python
+from sys import byteorder, stdin
+```
+
+
+### W1503: Redundant unittest assert {#W1503}
+
+The first argument of assertTrue and assertFalse is a "condition", which should
+evaluate to True or False. If the condition is a constant value, then it is
+considered to always be True, since it cannot be anything different.
+
+The warning message looks like:
+Redundant use of (assertTrue or assertFalse) with constant value <your-constant>
+
+```python
+import unittest
+
+class TestStringMethods(unittest.TestCase):
+    """Simple tests for example purposes."""
+
+    def test_isupper(self):
+        """Simple tests for example purposes."""
+
+        # Valid:
+        self.assertTrue('FOO'.isupper())
+        self.assertFalse('bar'.isupper())
+
+        # If a constant is passed as parameter, that condition is always true:
+        self.assertTrue('YES')
+        self.assertTrue(1)
+        self.assertTrue(True)
+        self.assertTrue(False)
+```
+
+
+### R0913: Too many arguments {#R0913}
+
+When too many arguments are passed into a function or method.
```

Comment:
"passed" is for calls. Just say something like "function/method is defined with too many arguments."